packages/core/src/common/utils/find-key-contains.injectable.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+
+export type FindKeyContains = <T>(map: Map<string, T>, text: string) => T | undefined;
+
+const findKeyContainsInjectable = getInjectable({
+  id: "find-key-contains",
+
+  instantiate: (): FindKeyContains => {
+    return <T>(map: Map<string, T>, text: string) => {
+      const entries = map.entries();
+
+      for (const [key, value] of entries) {
+        if (key.includes(text)) {
+          return value;
+        }
+      }
+
+      return undefined;
+    };
+  },
+});
```

Comment:
nit: This doesn't need to be an injectable because it has no side effects and has no dependencies.

Reply:
the file has been removed